localisation/00_zuwc_united_localisation.csv

```diff
@@ -2977,6 +2970,36 @@ text_effect_killing_speed_score;Plot Speed;;Verschw
+EVTOPTA_WCDAC_5040;Tendrils of the Twilight's Hammer will spread from here.;;Von hier aus wird sich der Zwielichthammer ausbreiten.;;;;;;;;;;;x
+EVTDESC_WCDAC_5041_first_kill;Your guards have found one of your followers dead this morning. Upon closer inspection of the body you realised that it was not the result of brawl getting out of hand, but an assassin's work. The cut was made with a precision of a master killer, but even more frightening is the fact, that you have seen cuts like this many times during your time with the Shadow Council.;;Eure Wachen haben heute Morgen einen Eurer Anh�nger tot aufgefunden. Bei n�herer Untersuchung der Leiche wurde klar, dass es sich nicht um das Ergebnis einer au�er Kontrolle geratenen Schl�gerei handelte, sondern um das Werk eines Attent�ters. Der Schnitt wurde mit der Pr�zision eines meisterhaften M�rders ausgef�hrt, aber noch be�ngstigender ist die Tatsache, dass Ihr derartige Schnitte w�hrend Eurer Zeit beim Schattenrat schon viele Male gesehen habt.;;;;;;;;;;;x
+EVTDESC_WCDAC_5041_later_kills;This morning your guards found another dead body inside your camp. The cut on the victim's throat was the same as the previous corpse and there could be no mistaking it for anything other than Garona's work. If the attacks continue, the success of your mission could be compromised.;;Heute Morgen fanden Eure Wachen eine weitere Leiche in Eurem Lager. Der Schnitt an der Kehle des Opfers war derselbe wie bei der vorherigen Leiche, und man kann ihn nicht mit etwas anderem als dem Werk Garonas verwechseln. Wenn die Angriffe weitergehen, k�nnte der Erfolg Eurer Mission gef�hrdet sein.;;;;;;;;;;;x
+EVTOPTA_WCDAC_5041;I know that blade... Garona.;;Ich kenne diese Klinge... Garona.;;;;;;;;;;;x
+EVTOPTB_WCDAC_5041;I need to stop them.;;Ich muss sie aufhalten.;;;;;;;;;;;x
+EVTDESC_WCDAC_5042;After some time spent planning you have finally created a scheme, that would allow you to capture Garona. Despite her mastery of assassination, there is one thing that makes her vulnerable. She has always greatly desired to hunt you down and kill you. Using yourself as a living bait would allow your followers to corner and capture her.;;Nach einiger Zeit der Planung habt Ihr endlich einen Plan entwickelt, der es Euch erlauben w�rde, Garona zu schnappen. Trotz ihrer meisterhaften Beherrschung des Attentats gibt es eine Sache, die sie verwundbar macht. Sie hat immer den gro�en Wunsch gehabt, Euch zur Strecke zu bringen und zu t�ten. Sich selbst als lebendigen K�der zu benutzen, w�rde es Euren Anh�ngern erm�glichen, sie in die Enge zu treiben und zu fangen.;;;;;;;;;;;x
```

Comment:
```suggestion
EVTDESC_WCDAC_5042;After some time spent planning you have finally created a scheme that will allow you to capture Garona. Despite her mastery of assassination, there is one thing that makes her vulnerable. She has always greatly desired to hunt you down and kill you. Using yourself as bait would allow your followers to corner and capture her.;;Nach einiger Zeit der Planung habt Ihr endlich einen Plan entwickelt, der es Euch erlauben w�rde, Garona zu schnappen. Trotz ihrer meisterhaften Beherrschung des Attentats gibt es eine Sache, die sie verwundbar macht. Sie hat immer den gro�en Wunsch gehabt, Euch zur Strecke zu bringen und zu t�ten. Sich selbst als lebendigen K�der zu benutzen, w�rde es Euren Anh�ngern erm�glichen, sie in die Enge zu treiben und zu fangen.;;;;;;;;;;;x
```